code/modules/power/apc.dm

```diff
@@ -67,6 +67,7 @@
 	var/cell_type = /obj/item/weapon/cell/apc
 	var/opened = 0 //0=closed, 1=opened, 2=cover removed
 	var/shorted = 0
+	var/night_mode = 0 // Determines if the light level is set to dimmed or not

```

Comment:
This is a bad implementation imo.

I'd say instead, implement a luminosity limit var, in the range 0-1
Luminosity of the light can be limited to normal value * that limit

This allows more finegrained ingame control of the lights